src/ruckig/position-second-step1.cpp

```diff
@@ -46,6 +48,11 @@ void PositionSecondOrderStep1::time_none(ProfileIter& profile, double vMax, doub
             }
         }
 
+        profile->t[3] = 0;
```

Comment:
Solution 1, can increment the iterator. `profile` is an iterator that might be modified in `Solution 1` so I believe to be sure the initialization of 3 - 6 should be done here, or alternatively could also be done in the function that increments the profile iterator.